### PR TITLE
DOC-975: Update Slack credential to High priority template

### PR DESCRIPTION
**Pull request by freakwriter**
Also update Slack node and Slack Trigger node

### CODE REVIEW

**Comment by freakwriter** [comment]
@Joffcom could I get you to review this PR? I started out just trying to update the credential but ended up touching the app node and trigger node since I was researching the methods/scopes anyway. I'm a bit on the fence about whether to show the required scopes on the credential page itself or to keep them in the separate node pages...anyway, I want to be sure the node documentation itself is correct, and if you have opinions on the scope stuff, I'm all ears, since I think I wrote myself in circles on this.

File: docs/integrations/builtin/app-nodes/n8n-nodes-base.slack.md [hunk 1]
```diff
@@ -17,53 +18,54 @@ Refer to [Slack credentials](/integrations/builtin/credentials/slack/) for guida
...
+* **Star**
+    * **Add** a star to an item
+    * **Delete** a star from an item
+    * **Get Many**: Get a list of an authenticated user's stars
+* **User**
+    * **Get** information about a user.
+	* **Get Many**: Get a list of users.
+    * **Get User's Profile**
+    * **Get User's Status**
+	* **Update User's Profile**
```
Comment (ayatnkw) [inline docs/integrations/builtin/app-nodes/n8n-nodes-base.slack.md:62]:
```suggestion
    * **Add** a star to an item.
    * **Delete** a star from an item.
    * **Get Many**: Get a list of an authenticated user's stars.
* **User**
    * **Get** information about a user.
	* **Get Many**: Get a list of users.
    * **Get User's Profile**.
    * **Get User's Status**.
	* **Update User's Profile**.
```

File: docs/integrations/builtin/credentials/slack.md [hunk 2]
```diff
@@ -21,8 +21,12 @@ You can use these credentials to authenticate the following nodes:
...
 ## Supported authentication methods
 
-- API access token
-- OAuth2
+- API access token:
+    - Required for the [Slack Trigger](/integrations/builtin/trigger-ndoes/n8n-nodes-base.slacktrigger/) node.
+    - Works with the [Slack](/integrations/builtin/app-nodes/n8n-nodes-base.slack/) node, but not recommended.
+- OAuth2:
+    - Recommended method for the [Slack](/integrations/builtin/app-nodes/n8n-nodes-base.slack/) node.
+    - Doesn't work with the [Slack Trigger](/integrations/builtin/trigger-ndoes/n8n-nodes-base.slacktrigger/) node.
```
Comment (ayatnkw) [inline docs/integrations/builtin/credentials/slack.md:29]:
```suggestion
    - Required for the [Slack Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.slacktrigger/) node.
    - Works with the [Slack](/integrations/builtin/app-nodes/n8n-nodes-base.slack/) node, but not recommended.
- OAuth2:
    - Recommended method for the [Slack](/integrations/builtin/app-nodes/n8n-nodes-base.slack/) node.
    - Doesn't work with the [Slack Trigger](/integrations/builtin/trigger-nodes/n8n-nodes-base.slacktrigger/) node.
```

Comment (ayatnkw) [reply docs/integrations/builtin/credentials/slack.md:29]: fixing url typo

**Review by ayatnkw — CHANGES_REQUESTED** [review]
Made a formatting suggestion and fixed a url typo. Also, I noticed a typo in the UI — if you notice something like this while working on the docs, could we maybe start listing them [here](https://linear.app/n8n/issue/DOC-1003/ui-fixes-typos-noticed-while-working-on-the-docs-overhaul)? @freakwriter

**Comment by Joffcom** [comment]
@freakwriter so I think I would put the scopes at the bottom of the doc maybe in a table as the same scopes will be needed for both oauth and app tokens so we can just link to the table from both.

for OAuth the scopes we will request are below if that helps as well.

```
	'channels:read',
	'channels:write',
	'chat:write',
	'files:read',
	'files:write',
	'groups:read',
	'im:read',
	'mpim:read',
	'reactions:read',
	'reactions:write',
	'stars:read',
	'stars:write',
	'usergroups:write',
	'usergroups:read',
	'users.profile:read',
	'users.profile:write',
	'users:read',
```

**Comment by freakwriter** [comment]
> Made a formatting suggestion and fixed a url typo. Also, I noticed a typo in the UI — if you notice something like this while working on the docs, could we maybe start listing them [here](https://linear.app/n8n/issue/DOC-1003/ui-fixes-typos-noticed-while-working-on-the-docs-overhaul)? @freakwriter

Yup! I haven't had a good place to capture those so I'm happy to log them wherever.

**Comment by freakwriter** [comment]
Thanks, @Joffcom , I'm pushing some changes to address this. I appreciate the feedback!

**Review by nik8n — APPROVED** [review]

**Review by Joffcom — APPROVED** [review]

**Review by ayatnkw — APPROVED** [review]

**Pull request merged**